content/blog/tikv-community-connections.md

```diff
@@ -0,0 +1,19 @@
+---
+title: TiKV Community Connections
+date: 2019-04-03
+author: Ana Hobden, TiKV Ecosystem Lead
+---
+
+A few months ago at [TiDB DevCon](https://pingcap.com/community-cn/devcon2019/) we talked about how we'd be opening new TiKV community forums & a community chat.

```

Comment:
Since the next paragraph is short says roughly the same, I combined them!

![image](https://user-images.githubusercontent.com/130903/55585420-73ac2980-56db-11e9-8579-e491e898d296.png)